### PR TITLE
feat: add scripts and docs

### PR DESCRIPTION
**Pull request by tzssangglass**

### CODE REVIEW

File: deploy/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    echo "deb http://openresty.org/package/debian $codename openresty" \ | sudo tee /etc/apt/sources.list.d/openresty.list
+    sudo apt-get update
+    sudo apt-get -y install openresty
+}
+
+start_openresty() {
+    ulimit -n 65535
+    ulimit -n -S
+    ulimit -n -H
+    openresty -p ~/apisix-benchmark/deploy/openresty -c ~/apisix-benchmark/deploy/openresty/conf/nginx.conf
```
Comment (spacewander) [inline deploy/deploy.sh:55]: We can avoid hardcoding the test path? Maybe we can use env var for it

File: deploy/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+
+deploy_apisix() {
+    if ! command_exists docker; then
+        install_docker
+    fi
+    sudo docker pull apache/apisix:dev
```
Comment (spacewander) [inline deploy/deploy.sh:63]: Ditto

**Review by spacewander — APPROVED** [review]

File: deploy/deploy.sh (hunk 1 above)
Comment (tzssangglass) [reply deploy/deploy.sh:55]: In fact, I hope to complete this optimization in the future, see the TODO in `Readme.md`.

**Comment by soulbird** [comment]
It is better to use a meaningful name instead of case1, case2.....

**Comment by soulbird** [comment]
https://github.com/apache/apisix-docker#how-to-run-apisix-in-stand-alone-mode

**Comment by tzssangglass** [comment]
> It is better to use a meaningful name instead of case1, case2.....

I originally used names like one-route, one-route-one-plugin, etc., but later changed them to case1, case2, etc. This was done because I wanted the documentation to reflect why we designed the test cases the way we did and what scenarios we wanted to verify. 

Anyway, I intend to update the Readme.md to introduce the test cases after the documentation is publicized, and changing the names alone will not convey the purpose of the test cases.

**Pull request merged**